### PR TITLE
[ci] remove systemtest englishbreakfast verilator tests

### LINKED ISSUE #12445: [bazel] update CI to run chip-level tests using bazel (over system test)
Currently CI only builds and runs host-side tests (e.g., DIF and mask ROM unit tests). CI should run all applicable (on-device) chip-level tests on FPGA and Verilator. This **_may_** include:

- [x] update test tags to define set of "fast" verilator chip-level tests (e.g. smoke tests) that match what is run with the current system test framework
    - [x] earlgrey tests
    - [x] englishbreakfast tests
        - [x] add to CI (#12783)
        - [x] remove old systemtest job from CI once above  is stable with bazel  (#12925)
- [x] run all (non-broken) chip-level tests in CI on FPGA and all (non-broken) fast chip-level tests on Verilator

### PR DESCRIPTION
**Pull request by timothytrippel**
In #12783, a CI job to build and test SW on the englishbreakfast
top-level Verilator sim using bazel was added. Now that this is stable,
this removes the equivalent systemtest CI.

This fixes #12445.

Signed-off-by: Timothy Trippel <ttrippel@google.com>

### CODE REVIEW

File: azure-pipelines.yml
Comment (rswarbrick) [inline azure-pipelines.yml:174]: This TODO message is specifically about how we hack the software. I don't *think* it applies to the `chip_englishbreakfast_verilator` target?

**Review by rswarbrick — APPROVED** [review]
LGTM, modulo a possible question about a comment.

File: azure-pipelines.yml
Comment (timothytrippel) [reply azure-pipelines.yml]: oops. copy/paste error. thanks!

**Review by milesdai — APPROVED** [review]

**Comment by timothytrippel** [comment]
`chip_englishbreakfast_cw305` relies on `sw_build_englishbreakfast` so this change cannot go through until #12953 fixed. Marking as draft until then.

File: azure-pipelines.yml
Comment (drewmacrae) [inline azure-pipelines.yml:146]: Maybe: This is true and correct but it's probably just going to create a merge conflict with https://github.com/lowRISC/opentitan/pull/12695

**Review by drewmacrae — APPROVED** [review]
Adding and then removing targets does make these neater. Looks like a bit of cleanup leaked into this too, but it's generally good.

File: azure-pipelines.yml
Comment (timothytrippel) [reply azure-pipelines.yml:146]: No worries, this will need to be rebased on those changes any way so I will wait until #12695 merges.

**Pull request merged**
